packages/cli-hydrogen/src/cli/commands/hydrogen/init.ts

```diff
@@ -0,0 +1,44 @@
+import {init} from '../../services/init.js'
+import {hydrogenFlags} from '../../flags.js'
+import {Flags} from '@oclif/core'
+import {cli, path} from '@shopify/cli-kit'
+import Command from '@shopify/cli-kit/node/base-command'
+
+export default class Build extends Command {
+  static description = 'Allows the quick initialisation of a Hydrogen storefront config file.'
+  static flags = {
+    ...cli.globalFlags,
+    ...hydrogenFlags,
+    country: Flags.string({
```

Comment:
Can we remove this flag as well as the `lang` one? let's scaffold the most minimal config possible.

https://github.com/Shopify/hydrogen/blob/v1.x-2022-07/templates/hello-world/hydrogen.config.ts#L3

Reply:
Done!